templates/committee_manager/edit.html.twig

```diff
@@ -14,6 +14,16 @@
 {% block committee_content %}
     <main>
         <section class="l__wrapper--slim">
+            {% if committee.isPending %}
```

Comment:
`if not approved` will be more correct as a committee can be in `pending` or in `pre-pending` status.